### DISCUSSION TITLE
Can't auto insert text indent after enter a new line inside template

### DISCUSSION THREAD

**Discussion by yangbo5207** [vue-vine/vue-vine#143] [Ideas]
![scroll](https://github.com/user-attachments/assets/f450f2a8-eeaa-411a-a97b-495ced3b9b67)

如图所示，我希望的效果是换行之后能自动缩进。

不知道这个跟 emmet 配置有关，还是需要什么，我的 emmet 配置如下所示

```js
"emmet.includeLanguages": {
  "javascript": "javascriptreact",
  "vue-html": "html",
  "wxml": "html",
  "typescript": "javascriptreact",
  "markdown": "javascriptreact",
  "mdx": "javascriptreact",
  "ts": "javascriptreact"
},
```

**Comment by ShenQingchuan** [comment]
是的，关于自动格式化这个问题，我们目前还在探索一个比较好的方案去做。

---
Yes, we're still researching a good way to implement code formatting.

**Comment by ShenQingchuan** [comment]
目前我们暂时没有特别多的精力来推进这个自动格式化的方案设计，因此暂时先将该问题转为了一个讨论帖，如果有同学有兴趣参与可以在这里回复。

Currently, we don't have much time to push forward this automatic formatting scheme design, so let's temporarily turn this issue into a discussion post. If anyone is interested, please feel free to reply here.

**Comment by ShenQingchuan** [comment]
@yangbo5207 我们目前提供了一套基于 Prettier 的格式化方案，集成在我们的 ESLint 配置中 `@vue-vine/eslint-config`。

伴随 VSCode 的自动修复功能，可以保证无论你随便写成什么样，最后都会格式化成统一的样子。

因此我理解这个问题目前已经暂时算解决了，后续我们会再开一个单独的讨论线程来延伸如何客制化格式化的方案，我们先暂时在这里关闭这个讨论帖。

---

We currently provide a format scheme based on Prettier that is integrated into our ESLint configuration `@vue-vine/eslint-config`.

With VSCode's autofix feature, you can ensure that no matter what you write, it will eventually format into a unified form.

Therefore, I understand that this problem has been temporarily solved, and we will open a separate discussion thread in the future to extend how to customize the format scheme. We will close this discussion thread here for the time being.
